manifests/init.pp

```diff
@@ -60,6 +60,7 @@
   $install_dir           = $::puppet_agent::params::install_dir,
   $disable_proxy         = false,
   $install_options       = $::puppet_agent::params::install_options,
+  $skip_if_unavailable   = 'absent',
```

Comment:
Seems like this would be easy to do with the splat operator, but we'd have to wait until Puppet 3 support is dropped.

CLA signed.